parser/src/main/java/io/moderne/jenkins/ingest/Merger.java

```diff
@@ -0,0 +1,78 @@
+package io.moderne.jenkins.ingest;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class Merger {
+
+    public static void main(String[] args) throws IOException {
+        mergeDatatables(Path.of(args[0]), Path.of(args[1]));
+    }
+
+    public static void mergeDatatables(Path original, Path newCsv) throws IOException {
+        Map<Key, CsvRow> newRows = parseNewCsv(newCsv);
```

Comment:
have you forgotten to add the CsvRow class?

Reply:
I've reused this record https://github.com/moderneinc/jenkins-ingest/blob/ebfc2628a6ac4d7d07a3eb0e0d1a57e4b0b45718/parser/src/main/java/io/moderne/jenkins/ingest/Parser.java#L113-L114